cloudbreak/cmd/cluster.go

```diff
@@ -100,7 +100,7 @@ func init() {
 							{
 								Name:   "new-network",
 								Usage:  "creates an aws cluster JSON template with new network",
-								Before: cf.CheckConfigAndCommandFlags,
+								Before: fl.CheckRequiredFlagsAndArguments,
```

Comment:
As I see there are optional flags like blueprint name? I think I was wrong and we go to the backend when the blueprintname is provided.